hek/handler.py

```diff
@@ -169,10 +170,11 @@ def get_nodes_by_paths(self, paths, node, cond=lambda x, y: True):
         return ()
 
     def get_def_id(self, filepath):
-        if not filepath.startswith('.') and '.' in filepath:
-            ext = os.path.splitext(filepath)[-1].lower()
-        else:
-            ext = filepath.lower()
+        filepath = Path(filepath)
+
+        ext = filepath.suffix.lower()
+        if len(ext) > 1:
+            ext = ext[1:]

```

Comment:
This is how you comment on part of the code while keeping it manageable

Reply:
And it means you can actually reply to it!

Reply:
This by the way, if you tested the tools, would have been a clear sign that `open()` can handle path objects. Because that code snippet had a 100% failure rate for getting the extension. Which means that for the past month or so all my tag types have been detected by the code below.